sh/upload.sh

```diff
@@ -82,11 +82,11 @@ _update() {
     [ "${GLOBAL_INSTALL}" = true ] && ! [ "$(id -u)" = 0 ] && printf "%s\n" "Error: Need root access to update." && return 0
     [ "${job_update}" = uninstall ] && job_uninstall="--uninstall"
     _print_center "justify" "Fetching ${job_update} script.." "-"
-    repo_update="${REPO:-labbots/google-drive-upload}" type_value_update="${TYPE_VALUE:-latest}"
+    repo_update="${REPO:-labbots/google-drive-upload}" type_value_update="${TYPE_VALUE:-latest}" cmd_update="${COMMAND_NAME:-gupload}" path_update="${INSTALL_PATH:-${HOME}/.google-drive-upload/bin}"
     { [ "${TYPE:-}" != branch ] && type_value_update="$(_get_latest_sha release "${type_value_update}" "${repo_update}")"; } || :
     if script_update="$(curl --compressed -Ls "https://github.com/${repo_update}/raw/${type_value_update}/install.sh")"; then
         _clear_line 1
-        printf "%s\n" "${script_update}" | sh -s -- ${job_uninstall:-} --skip-internet-check
+        printf "%s\n" "${script_update}" | sh -s -- ${job_uninstall:-} --skip-internet-check --cmd "${cmd}" --path "${path_update}"
```

Comment:
Codacy found an issue: [cmd is referenced but not assigned.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6363604)